dts/bindings/mtd/partition.yaml

```diff
@@ -16,6 +16,13 @@ properties:
 child-binding:
     description: Flash partition child node
     properties:
+       mount-type:
+          type: string
+          required: false
+          description: file system identifier, e.g. "LittleFS"
+       mount-point:
```

Comment:
should be `zephyr,mount-point`.  Please also add a description about what this is.